docs/sources/shared/datasources/tempo-traces-to-profiles.md

```diff
@@ -94,9 +75,23 @@ To use a custom query with the configuration, follow these steps:
+1.  Specify a custom query to be used to query profile data. You can use various variables to make that query relevant for current span. The link shows only if all the variables are interpolated with non-empty values to prevent creating an invalid query. You can interpolate the configured tags using the `$__tags` keyword.
 1.  Select **Save and Test**.
+
+## Configuration options
+
+The following table describes options for configuring your Trace to profiles settings:
+
+| Setting name         | Description                                                                                                                                                                                                                                                                                                    |
+| -------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| **Data source**      | Defines the target data source. You can select a Pyroscope \[profiling\] data source.                                                                                                                                                                                                                          |
+| **Tags**             | Defines the tags to use in the profile query. Default: `cluster`, `hostname`, `namespace`, `pod`, `service.name`, `service.namespace`. You can change the tag name for example to remove dots from the name if they're not allowed in the target data source. For example, map `http.status` to `http_status`. |
+| **Profile type**     | Defines the profile type that used in the query.                                                                                                                                                                                                                                                               |
+| **Use custom query** | Toggles use of custom query with interpolation.                                                                                                                                                                                                                                                                |
+| **Query**            | Input to write custom query. Use variable interpolation to customize it with variables from span.                                                                                                                                                                                                              |
```

Comment:
Don't bold these UI elements because they're the only thing in the table cell.

Reply:
Even though these are UI elements?

Reply:
Fixed.